packages/back-end/src/models/ExperimentModel.ts

```diff
@@ -286,6 +287,14 @@ export async function updateExperimentById(
       $set: changes,
     }
   );
+
+  const updated = await getExperimentById(organization, experiment.id);
```

Comment:
We don't need to re-fetch from the database.  You have the original `experiment` object and `changes`.  So you can just do 

```ts
const updated = {
  ...experiment,
  ...changes
}
```